### ISSUE TITLE
Bug with deco function call

### ISSUE THREAD

**Issue by AlysonR** [alex-sherman/deco#56]
The following example seems to not work:
```python
@concurrent
def get_region(region_filepath, variables_to_get):
	return {}

@synchronized
def get_data(variable_list, regions):
	return_variables = {}
	for key in variable_list:
		return_variables[key] = []
	region_dicts = []
	for lat_lon in regions:
		region_dicts.append(get_region(root_dir + lat_lon, variable_list))
	#get_region.wait()
	print("len", len(region_dicts), regions)
	#consolidate all the regions
	for region in region_dicts:
		for key in region.keys():
			return_variables[key].extend(region[key])
	return return_variables
```

**Comment by alex-sherman** [comment]
Fixed in 04f13bf1dc2981416bae96e31d3eb1629661ba88

**Comment by AlysonR** [comment]
🥰🥰🥰🥰
